app/lib/report_helper.rb

```diff
@@ -103,15 +122,15 @@ def sort_report!(nt_zscores, sort_by)
 
   def compute_rpm(count, total_reads)
     if count
-      count * 1e6 / total_reads.to_f
+      (count * 1e6 / total_reads.to_f).round(3)
```

Comment:
Should rounding happen here or in JS?   I see advantages to both.  Note, however, this isn't the only place that computes zscores, so you can't rely on the other zscores merged from background summary to be rounded the way you like.   This would be an argument to push rounding into JS.